user_guide_src/source/installation/upgrade_420.rst

```diff
@@ -24,7 +24,7 @@ The following files received significant changes and
 Breaking Changes
 ****************
 
-
+- The ``system/bootstrap.php`` file no longer returns a ``CodeIgniter`` instance, and does not load the ``.env`` file (now handled in ``index.php`` and ``spark``). If you have code that expects these behaviors it will no longer work and must be modified.
```

Comment:
Please mention _why_ this change was made as well.

Reply:
Done.